Spigot-Server-Patches/0282-Make-player-data-saving-configurable.patch

```diff
@@ -0,0 +1,35 @@
+From f3a00869c67d739b55595be61852de56d3faae8e Mon Sep 17 00:00:00 2001
+From: Mark Vainomaa <mikroskeem@mikroskeem.eu>
+Date: Mon, 26 Mar 2018 18:30:53 +0300
+Subject: [PATCH] Make player data saving configurable
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/PaperConfig.java b/src/main/java/com/destroystokyo/paper/PaperConfig.java
+index ec89ecfca..d17862430 100644
+--- a/src/main/java/com/destroystokyo/paper/PaperConfig.java
++++ b/src/main/java/com/destroystokyo/paper/PaperConfig.java
+@@ -282,4 +282,9 @@ public class PaperConfig {
+     private static void authenticationServersDownKickMessage() {
+         authenticationServersDownKickMessage = Strings.emptyToNull(getString("messages.kick.authentication-servers-down", authenticationServersDownKickMessage));
+     }
++
++    public static boolean playerDataSavingDisabled = false;
++    private static void playerDataSavingDisabled() {
++        playerDataSavingDisabled = getBoolean("settings.player-data-saving-disabled", playerDataSavingDisabled);
```

Comment:
this being inverted is confusing imo. I would rather it be "settings.save-player-data: true"

Reply:
Done, I flipped the naming